### PR TITLE
User authentication

### PR DESCRIPTION
**Pull request by MattSkrobis**

### CODE REVIEW

File: app/components/user-form/component.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import lookupValidator from 'ember-changeset-validations';
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
+  paperToaster: service(),
+  currentUser: service(),
+  router: service(),
+  store: service(),
+  genders: ['male', 'female'],
```
Comment (PoslinskiNet) [inline app/components/user-form/component.js]: Translation is missing

File: app/components/user-form/component.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
+  paperToaster: service(),
+  currentUser: service(),
+  router: service(),
+  store: service(),
```
Comment (PoslinskiNet) [inline app/components/user-form/component.js]: Redundant

File: app/controllers/login.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+const { Controller, inject: { service } } = Ember;
+
+export default Controller.extend({
+  session: service('session'),
```
Comment (PoslinskiNet) [inline app/controllers/login.js]: Use `service()` only. The Same in the next line.

File: app/routes/login.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
```
Comment (PoslinskiNet) [inline app/routes/login.js]: This file is redundant (the same route will be automatically generated by Ember).

File: app/components/user-form/component.js (hunk 1 above)
Comment (MattSkrobis) [reply app/components/user-form/component.js]: It is not: https://github.com/MattSkrobis/bsc-ember/pull/31/files/a5fe7e3b7198b88083cae2e65972f7d2feb45f3c#diff-a448ae814c0d31ba194bde0eb91f8b6bR13

**Comment by MattSkrobis** [comment]
@PoslinskiNet feedback resolved

**Pull request merged**